### PR TITLE
fix: Fix android statusbar translucency

### PR DESCRIPTION
**Pull request by DragonSpirit**
Sup!
This PR should fix transparency issue from #321. It would be great if @vikrantnegi or anybody else can check this.

### CODE REVIEW

**Comment by vikrantnegi** [comment]
@DragonSpirit Hey, I just reviewd the PR on my Android and unfortunately it is not seems to working as expected.

However, when I added `barStyle` to the `StatusBar` it worked.

```
<StatusBar translucent={false} barStyle="dark-content" />
```

cc @amaurymartiny

**Comment by DragonSpirit** [comment]
@vikrantnegi I am not sure it's correct way for light-based theme devices. Main point is disable transparency from statusbar. Can you attach screenshot for check difference?

**Comment by vikrantnegi** [comment]
Good point. But we can always update the bar style as per the app theme like this: 

```
<StatusBar
  barStyle={
    this.state.theme === 'light'
      ? 'dark-content'
      : 'light-content'
  }
  backgroundColor={
    this.state.theme === 'light' ? '#fff' : '#000'
  }
/>
```
I [implemented](https://github.com/vikrantnegi/RN-Book-Search/blob/master/App.js#L35) this in one of my expo apps.

Here is the screenshots before and after adding the `barStyle`.
Before:
Before-barStyle

After:
After-barStyle

**Comment by DragonSpirit** [comment]
@vikrantnegi, thank you for research. But I think theme feature for statusbar should be implemented in #320. Use force dark-content can break iOS dark theme.
@amaurymartiny what do you think about it?

**Comment by vikrantnegi** [comment]
@DragonSpirit Using `dark-content` won't break iOS dark theme as the app doesn't support dark theme, so iOS won't apply dark theme to the app.

File: App/Screens/Screens.tsx [hunk 1]
```diff
@@ -139,5 +139,10 @@ export function Screens(): React.ReactElement {
 
   const stack = renderScreen(api, error);
 
-  return <View style={theme.fullScreen}>{stack}</View>;
+  return (
+    <View style={theme.fullScreen}>
+      <StatusBar translucent={false} />
```
Comment (amaurymartiny) [inline App/Screens/Screens.tsx]: Could you actually put this in `App.tsx`?

Comment (DragonSpirit) [reply App/Screens/Screens.tsx]: Sure, I'm done with it yet

**Comment by amaurymartiny** [comment]
Thanks a lot @DragonSpirit and @vikrantnegi!

**Pull request merged**